src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.Random.cs

```diff
@@ -3,10 +3,10 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
+        private string generateInstancePrefix()
         {
             byte[] bytes = new byte[8];
-            s_random.Value.NextBytes(bytes);
+            s_random.NextBytes(bytes);
```

Comment:
There is no high level locking.    I was never thrilled with using Random here (introduces a static etc).   My recommendation is to remove Random and the byte stuff, and instead simply use Guid.NewGuid() (which also generates a random number) and then pull out 64 bits of it.